go/vt/vttablet/onlineddl/executor.go

```diff
@@ -1146,7 +1188,7 @@ func (e *Executor) validateAndEditAlterTableStatement(ctx context.Context, onlin
 			}
 		case *sqlparser.AddConstraintDefinition:
```

Comment:
OK, so I take it that this matches FK and CHECK constraints?

Reply:
That's right!